components/NavigationBar/tests/unit/NavigationBarTests.m

```diff
@@ -199,6 +200,32 @@ - (void)testTitleFontProperty {
   XCTAssertEqual(fontTraits, resultTraits);
 }
 
+- (void)testTitleFontPropertyWithAllowAnyTitleFontSizeEnabled {
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
+  navBar.frame = CGRectMake(0, 0, 300, 25);
+  navBar.title = @"this is a Title";
+  navBar.titleAlignment = MDCNavigationBarTitleAlignmentCenter;
+  navBar.allowAnyTitleFontSize = YES;
+  [navBar layoutIfNeeded];
+
+  XCTAssertNotNil(navBar.titleFont);
+  XCTAssertEqual(navBar.titleLabel.font, navBar.titleFont);
+
+  UIFont *font = [UIFont systemFontOfSize:24];
+  navBar.titleFont = font;
+
+  UIFont *resultFont = navBar.titleLabel.font;
+  XCTAssertEqualObjects(resultFont.fontName, font.fontName);
+  XCTAssertEqualWithAccuracy(resultFont.pointSize, 24, 0.01);
+
+  NSDictionary <NSString *, NSNumber *> *fontTraits =
+      [[font fontDescriptor] objectForKey:UIFontDescriptorTraitsAttribute];
+  NSDictionary <NSString *, NSNumber *> *resultTraits =
+      [[resultFont fontDescriptor] objectForKey:UIFontDescriptorTraitsAttribute];
+
+  XCTAssertEqual(fontTraits, resultTraits);
```

Comment:
Please consider using `XCTAssertEqualObjects`.  Should this be the exact same object or equal objects?

Reply:
Done.